lib/experimental/webfonts.php

```diff
@@ -76,25 +74,68 @@ function wp_register_webfonts( array $webfonts = array() ) {
  * Example of how to register Source Serif Pro font with font-weight range of 200-900:
  *
  * If the font file is contained within the theme:
- * ```
+ *
+ * <code>
  * wp_register_webfont(
  *      array(
  *          'provider'    => 'local',
- *          'font_family' => 'Source Serif Pro',
- *          'font_weight' => '200 900',
- *          'font_style'  => 'normal',
+ *          'font-family' => 'Source Serif Pro',
+ *          'font-weight' => '200 900',
+ *          'font-style'  => 'normal',
  *          'src'         => get_theme_file_uri( 'assets/fonts/source-serif-pro/SourceSerif4Variable-Roman.ttf.woff2' ),
  *      )
  * );
- * ```
+ * </code>
  *
  * @since 6.0.0
  *
  * @param array $webfont Webfont to be registered.
- *                       See {@see WP_Webfonts_Registry::register()} for a list of supported arguments.
  */
 function wp_register_webfont( array $webfont ) {
-	wp_webfonts()->register_font( $webfont );
+	wp_webfonts()->register_webfont( $webfont );
+}
+
+/**
+ * Enqueues a collection of font families.
+ *
+ * Example of how to enqueue Source Serif Pro and Roboto font families, both registered beforehand.
+ *
+ * <code>
+ * wp_enqueue_webfonts(
+ *  'Roboto',
+ *  'Sans Serif Pro'
+ * );
+ * </code>
+ *
+ * Font families should be enqueued from the `init` hook or later.
+ *
+ * @since 6.0.0
+ *
+ * @param string[] $webfonts Font families to be enqueued.
+ */
+function wp_enqueue_webfonts( $webfonts ) {
+	foreach ( $webfonts as $webfont ) {
+		wp_enqueue_webfont( $webfont );
+	}
+}
+
+/**
+ * Enqueue a single font family that has been registered beforehand.
+ *
+ * Example of how to enqueue Source Serif Pro font:
+ *
+ * <code>
+ * wp_enqueue_webfont( 'Source Serif Pro' );
+ * </code>
+ *
+ * Font families should be enqueued from the `init` hook or later.
+ *
+ * @since 6.0.0
+ *
+ * @param string $webfont Font family to be enqueued.
+ */
+function wp_enqueue_webfont( $webfont ) {
+	wp_webfonts()->enqueue_webfont( $webfont );
 }
```

Comment:
Return the bool flag from `enqueue_webfont()` to provide an indication if the enqueue was successful or not.
And it change the function's parameter name and comment to match the signature of `WP_Webfonts::enqueue_webfont()`, i.e. enqueues by a font family name.

```suggestion
 * @param string $webfont Font family to be enqueued.
 */
function wp_enqueue_webfont( $webfont ) {
	wp_webfonts()->enqueue_webfont( $webfont );
}
```

```suggestion
 * @param string $font_family_name The font family name to be enqueued.
 * @return bool True if successfully enqueued, else false.
 */
function wp_enqueue_webfont( $font_family_name ) {
	return wp_webfonts()->enqueue_webfont( $font_family_name );
}
```

Reply:
Resolved in https://github.com/WordPress/gutenberg/pull/39559/commits/27e7da71a053c7e21c91d83935fbb1fd460f376b.